data/block/block.proto

```diff
@@ -59,6 +59,19 @@ message PeerChange {
   uint32 ShardIdDest = 2 [(gogoproto.jsontag) = "shardIdDest"];
 }
 
+// ExecutionResult defines the dto that will hold information about execution result
+message ExecutionResult {
+  bytes  HeaderHash                         = 1 [(gogoproto.jsontag) = "headerHash"];
+  uint64 Nonce                              = 2 [(gogoproto.jsontag) = "headerNonce"];
+  uint64 HeaderRound                        = 3 [(gogoproto.jsontag) = "headerRound"];
+  bytes  RootHash                           = 4 [(gogoproto.jsontag) = "rootHash"];
+  bytes  ReceiptRootHash                    = 5 [(gogoproto.jsontag) = "receiptRootHash"];
```

Comment:
maybe rename this as ReceiptsHash ?
we are not yet transitioning to a receipts trie

Reply:
done